components/inputs/docs/input-textarea.md

```diff
@@ -0,0 +1,81 @@
+# TextAreas
+
+The `<d2l-input-textarea>` is a wrapper around the native `<textarea>` element that provides auto-grow and validation behaviours. It's intended for inputting unformatted multi-line text.
+
+![example screenshot of text input](../screenshots/textarea.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-textarea.js';
+</script>
+<d2l-input-textarea
+  label="Description"
+  placeholder="Description of your topic."
+  value="Some description..."></d2l-input-textarea>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `aria-invalid` | String | Indicates that the `textarea` value is invalid |
+| `description` | String | A description to be added to the `textarea` for accessibility |
+| `disabled` | Boolean | Disables the `textarea` |
+| `label` | String, required | Label for the `textarea` |
+| `label-hidden` | Boolean | Hides the label visually (moves it to the `textarea`'s `aria-label` attribute) |
+| `max-height` | String, default: 12rem | Maximum height of the input before scrolling. `none` allows `textarea` to grow infinitely. |
+| `min-height` | String, default: none | Minimum height of the input. If `min-height` and `max-height` are equal then auto-grow will be disabled. |
```

Comment:
Thoughts about having the default min-height be multi-line, similar to how a default `<textarea>` would look? I'm worried that if it's single line it won't look like a textarea and folks will always be changing the default.

Reply:
Yeah that's a good question.  I'd be fine with it.  What do you think @geurts ?

Reply:
I'm fine with that too. Maybe 6rem would work?